README.md

```diff
@@ -23,3 +23,18 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+## Plagarism Statement
+
+All exercises must contain the following statement:
+“I certify that I have listed all sources used to complete this exercise, including the use
+of any Large Language Models. All of the work is my own, except where stated
+otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is
+suspected, charges may be filed against me without prior notice.”
+
+## Note
+I did not complete this excersise last semester, so I am attempting this completely from scratch. I did discuss this with Lily Brongo and then referenced Ishita Patel's repository from last semester to confirm my logic for the code and response before submission. For this one in particular, I went to her in person to help me through setting up the testing correcting some of my code. For additional clarity, there is one portion of the test called classGraph(Marked in test code) that is directly taken from Ishita's repository, it was used to show me the syntax that she used and I built from there.
```

Comment:
So what parts of the code are yours?

Reply:
I went through and marked everything that I had help with. The long and short of it is that the innermost if statement was something she sat at my computer and explained to me while she coded it and there were a few other logic things she explained to me, and I coded them. I marked these instances. Additionally, the test case she gave me was already commented so I assume that wasn't what you were asking about.